### DISCUSSION TITLE
International Branding

### DISCUSSION THREAD

**Discussion by kuipou** [Shopify/checkout-branding#2] [Feature requests]
From what I saw in both my partner account and my own store the brand don't have a possibility of internalisation as my brand is composed both English and french wordmarks and an emblem
what i could see happening is that the brand page detects if theres an other language present and ask if the merchant wants to use an addional logo for that language

**Comment by akatigbak** [comment]
thanks for the feedback on this! Always iterating on how we can make customizations easier so we'll take this back to the team for future consideration :)
